resources/public/api/conf/1.0/endpoints/charitableGiving_retrieve.raml

```diff
@@ -0,0 +1,27 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
+  - errors.ruleTaxYearNotSupported
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
```

Comment:
This description does not quite match the spec documentation

Reply:
Updated - now uses errors.notFoundCharitableGiving instead